astroquery/simbad/core.py

```diff
@@ -768,42 +768,52 @@ def query_region(self, coordinates, radius=2*u.arcmin, *,
         center = center.transform_to("icrs")
 
         top, columns, joins, instance_criteria = self._get_query_parameters()
+        if criteria:
+            instance_criteria.append(f"({criteria})")
 
         if center.isscalar:
             radius = coord.Angle(radius)
             instance_criteria.append(f"CONTAINS(POINT('ICRS', basic.ra, basic.dec), "
                                      f"CIRCLE('ICRS', {center.ra.deg}, {center.dec.deg}, "
                                      f"{radius.to(u.deg).value})) = 1")
+            return self._query(top, columns, joins, instance_criteria,
+                               get_query_payload=get_query_payload)
+
+        if len(center) > 200000:
```

Comment:
maybe a comment where this 200000 is coming from, link to the simbad docs or similar? (E.g. the number has changed from 10000 to 200000, would be nice to be able to track/have a reference for any future changes)

Reply:
10000 was random, it's something I did not change when we switched to TAP. I guess the person who wrote the former method had timeout issues?

200000 comes from SIMBAD's declared capabilities https://simbad.cds.unistra.fr/simbad/sim-tap/capabilities , under uploadLimit. I'm not sure if a link to the capabilities would help a lot the users, but I can definitely add a comment for the people reading the code

Reply:
There is now a new property `uploadlimit` read from the capabilities, so that if this changes upstream (I talked with my colleague and it happened in the past) then astroquery won't be impacted.

Reply:
Thank you! I meant this to be a code comment for the reader, but doing it with capabilities is indeed better. Thanks!